deployment/localsettings.template.py

```diff
@@ -15,3 +15,6 @@
 
 # Make apache work when DEBUG == False
 ALLOWED_HOSTS = ["localhost", "127.0.0.1"]
+
+# Questionnaires automatically added to exam evaluations
+EXAM_QUESTIONNAIRE_IDS = [83]
```

Comment:
It would be better to create a new "Exam" questionnaire in the test data that is used here.

Reply:
Shouldn't we come up with something nicer than hardcoding ids in the settings? The best idea I have so far would be to go via the title of the questionnaire, which still feels a bit hacky but better than the id to me

Reply:
I think I'd prefer the id as its constant. Any other field might change (if only hypothetically in the future after adding forms for that), so using them for identifing objects seems more brittle to me.

Reply:
Fair enough, my point was more about the UX of looking up database ids. We can revisit this later / in another issue though - or not at all if you disagree

Reply:
I see that having to look up IDs from browser urls is a bit awkward, but then again, editing the localsettings.py requires SSHing into the server and editing the right text file in the right location, so for me, they both rank at "for expert admin who knows what they're doing". I'm also fine with other fields in general, but I'd expect them to cause more problems if they are 1. not unique or 2. mutable. Afaik, the pk is the only unique constant field.

Reply:
I was thinking about something on the website, completely disconnected from the settings. I agree that for our current audience we are fine like this though